app/src/main/res/values/colors.xml

```diff
@@ -71,7 +71,7 @@
     <item name="litecoin_gray" type="color">#F1F1F1</item>
     <item name="litecoin_silver" type="color">#A6A9AA</item>
     <item name="litecoin_dark_silver" type="color">#4D4D4E</item>
-    <item name="litecoin_litewallet_blue" type="color">#345D9D</item>
+    <item name="litecoin_litewallet_blue" type="color">#0C3475</item>
```

Comment:
Sounds like we need to add litewalletDarkBlue to the resource. Any reason why we cant add it ? Right now it is not going to be a easy reference (Changing the color value)

Reply:
We can add